src/helpers/figures/charts/chart_common.ts

```diff
@@ -246,7 +248,11 @@ function createDataSet(
 /**
  * Transform a dataSet to a ExcelDataSet
  */
-export function toExcelDataset(getters: CoreGetters, ds: DataSet): ExcelChartDataset {
+export function toExcelDataset(
+  getters: CoreGetters,
+  ds: DataSet,
+  customDs?: CustomizedDataSet
```

Comment:
Why do we now need to give the customizedDataSet to this function ? How comes that before we were able to have the correct colors, labels and axes without passing customDs here ? Isn't it possible to do it the same way for the trend line ?

Reply:
@anhe-odoo I need the trend to be add to the generated excel dataset. Therefore, I need the trend to be passed here. I can pass only the trend as a function argument instead. What do you think ?